data/settings_mapping.json

```diff
@@ -285,29 +285,31 @@
       "sections": [
         {
           "name": "generalcosmetics_section",
-          "text": "General",
-          "row_span": 1,
+          "col_span": 12,
```

Comment:
@rattus128 Please change the col_span here from 12 to 4. Technically there are 12 columns now, but I opted to cap it off at 4 sections per row max, so I still work with 1-4 internally and then just multiply it for the new 12 layout. Currently it will ignore anything above 4, so this is not technically wrong, but for the correct spirit please change it to 4.